lightly/api/api_workflow_upload_dataset.py

```diff
@@ -22,7 +24,7 @@
 class _UploadDatasetMixin:
 
     def upload_dataset(self: ApiWorkflowClient, input: Union[str, LightlyDataset], max_workers: int = 8,
-                       mode: str = 'thumbnails', verbose: bool = True):
+                       size: Union[int, tuple] = -1,  mode: str = 'thumbnails', verbose: bool = True):
```

Comment:
rather Union[int, Tuple[int, int]]] and Tuple should be imported from typing.
Using types directly as typehints (i.e. tuple instead of tuple) is only supported in python 3.9.